conda_smithy/configure_feedstock.py

```diff
@@ -319,6 +322,7 @@ def main(forge_file_directory):
     recipe_dir = 'recipe'
     config = {'docker': {'image': 'condaforge/linux-anvil', 'command': 'bash'},
               'templates': {'run_docker_build': 'run_docker_build_matrix.tmpl'},
+              'ci_enabled': [],
```

Comment:
So I have updated this to use `<CI>.enabled` format. If there was a parameter, we definitely overwrite it with these new changes. Though maybe we should be issue a warning if a user tries to set this.